package.json

```diff
@@ -12,7 +12,10 @@
     "start": "npm run build",
     "watch": "webpack --watch",
     "deploy:docs": "./scripts/deploy.sh",
-    "run-tests": "skpm-test --app=xcode",
+    "run-tests": "skpm-test",
+    "run-tests-xcode": "skpm-test --app=xcode",
```

Comment:
instead of `testx` and such, can you create `test:no-variant` which is more explicit (see https://github.com/BohemianCoding/SketchAPI/pull/142/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15)? and also please keep the default `test` as the tests with xcode :)

Reply:
Yes, true.